src/NuGet.Core/NuGet.Packaging/PackageCreation/Resources/NuGetResources.resx

```diff
@@ -195,4 +195,17 @@
     <value>The licenseUrl and license elements cannot be used together.</value>
     <comment>Please don't localize licenseUrl and license</comment>
   </data>
+  <data name="IconCannotOpenFile" xml:space="preserve">
+    <value>Cannot open icon file '{0}': {1}.</value>
+    <comment>0 - Path to the icon file, 1 - Error message for not opening the icon</comment>
+  </data>
+  <data name="IconMaxFilseSizeExceeded" xml:space="preserve">
+    <value>The icon file size must not exceed 1 megabyte.</value>
+  </data>
+  <data name="IconErrorEmpty" xml:space="preserve">
+    <value>The icon file is empty.</value>
+  </data>
+  <data name="IconNoFileElement" xml:space="preserve">
+    <value>No icon &lt;file&gt; element found for &lt;icon&gt; in the nuspec.</value>
```

Comment:
How about including the path of the icon specified in the nuspec? 
And then change the message to something: 

>  The icon file '{0}' does not exist in the package.

analogous to what the license check says. We should keep the language similar

The icon file {0} could not be found in the package sounds better, but that'd require changing both (including docs). 
I am icon if you want to change them both, but that's just more work for you :)

Reply:
I will change it. Which part of the docs should I change ?

Reply:
I just searched online :)

https://docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5030

Reply:
Actually you are not changing it to the new suggestion, so it's fine, no need to update any docs :)